Cargo.toml

```diff
@@ -18,3 +18,11 @@ ring = { version = "0.16.20", features = ["std"] }
 serde = { version = "1.0.104", features = ["derive"] }
 serde_json = "1.0.78"
 thiserror = "1.0.30"
+
+[dev-dependencies]
+anyhow = "1.0.66"
+clap = { version = "4.0.29", features = ["derive"] }
```

Comment:
Looks like this will need an MSRV bump (we should also set the `rust-version` in the `package` metadata if it's > 1.56).

Reply:
Since these are dev-dependencies only used for the example, can we leave MSRV as-is?

Reply:
Makes CI a little tricky. Plus, the new MSRV is old enough at this point that it probably doesn't matter. The majority of the ecosystem sticks pretty close to stable.

Reply:
Bumped edition and rust-version